js/src/util/validation.js

```diff
@@ -42,9 +44,14 @@ export function validateAbi (abi) {
   try {
     abiParsed = JSON.parse(abi);
 
-    if (!util.isArray(abiParsed)) {
+    if (!apiutil.isArray(abiParsed)) {
```

Comment:
the underlying implementation is an issue. i can easily implement something that is not an array (not even iterable) but passes `apiutil.isArray`.

I'm suggesting you a API-compatible one-line change to the more robust, core JS `Array.isArray. I'm also fine with lodash `isArray`. I just see no reason to use homegrown check, especially if they're more brittle then the builtins.

I don't want to start an ideological war, this is only about reducing possible sources of errors.